src/Auth/wwwroot/js/signout-redirect.js

```diff
@@ -0,0 +1,6 @@
+﻿window.addEventListener("load", function () {
```

Comment:
Prefer `() => {` syntax (ES6) over `function () {`.

Reply:
There is no Babel here to convert the ES6 syntax in to ES5/ES3 supported syntax. There won't be old browser support.

Reply:
...mutter mutter old browsers